02_operation_of_running_systems/100_automation_with_scripts/101_shell_script.md

```diff
@@ -0,0 +1,146 @@
+# Shell Scripts
+This chapter is based on the chapter 11 of Linux - Rheinwerk Verlag.
+Shell scripts make it possible to write down complex tasks into scripts. These scripts can be used to automate management and orchestration.
+
+## Hello World!
```

Comment:
Your content is good, but you should split this big file into smaller ones.

Think of future students using your *.md files to learn shell scripting.

- May you split this file on the 2nd level ( ## ) into smaller chunks ?
- Could you provide for each level 2 unit an own example and an own task ?

Reply:
Done. I designed the exercises to build on each other.